.github/scripts/bot-advanced-check.sh

```
@@ -67,20 +68,21 @@ get_intermediate_count() {
+      query($owner: String!, $name: String!, $user: String!, $labels: [String!]!) {
         repository(owner: $owner, name: $name) {
           intermediate: issues(
             first: 1
             states: CLOSED
-            filterBy: { assignee: $user, labels: ["intermediate"] }
+            filterBy: { assignee: $user, labels: $labels }
           ) {
             totalCount
           }
         }
       }' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)